src/components/Todos/index.js

```diff
@@ -7,17 +7,41 @@ import deserialize from './storage-source';
 import serialize from './storage-sink';
 import TodoItem from '../TodoItem';
 
+// AMEND STATE WITH CHILDREN
+// This is a callback function for a map function used below.
```

Comment:
Not really. This function **creates** the callback (I wouldn't call it callback either, it's just a "projection" function) for the map function below.